shell/platform/android/io/flutter/Log.java

```diff
@@ -9,41 +9,52 @@
 import io.flutter.BuildConfig;
 
 /**
- * Port of {@link android.util.Log} that only logs in {@link BuildConfig#DEBUG} mode.
+ * Port of {@link android.util.Log} that only logs in {@link BuildConfig#DEBUG} mode and
+ * internally filters logs based on a {@link #logCutoff}.
  */
 public class Log {
+  private static int logCutoff = android.util.Log.DEBUG;
+
+  /**
+   * Sets a log cutoff such that a log level of lower priority than {@code logCutoff} is
+   * filtered out.
+   */
+  public static void setLogCutoff(int logCutoff) {
```

Comment:
nit: I think we use the term level (e..g `setLogLevel`) elsewhere for this concept.  If cutoff is more normal for Android then that's fine.

Reply:
I'll change to that